rotkehlchen/db/upgrades/v43_v44.py

```diff
@@ -96,6 +98,25 @@ def _addressbook_schema_update(write_cursor: 'DBCursor') -> None:
     fix_address_book_duplications(write_cursor=write_cursor)
 
 
+@enter_exit_debug_log()
+def _add_uniswap_to_historical_oracles(cursor: 'DBCursor') -> None:
+    """Add uniswapv2 and uniswapv3 to the list of historical price oracles."""
+    cursor.execute('SELECT value FROM settings WHERE name=?', ('historical_price_oracles',))
+    price_oracles = cursor.fetchone()
```

Comment:
If the variable isn't set I believe you don't need to do anything since it will use the default. I believe that if price_oracles is None you just exist the logic